tool/gh_actions/generate_documentation.sh

```diff
@@ -17,6 +17,9 @@ set -euo pipefail
 # The documentation will be placed in the "doc/api" folder.
 output=$(dart doc --validate-links 2>&1 | tee)
 
+# copy and replace the documentation generated to the website
+cp -rf doc/api/ doc/website/
```

Comment:
Need to update all the github.io links in README and in pubspec.yaml so that they now point to the right area

Reply:
Fixed.